lib/api/bucketPut.js

```diff
@@ -8,6 +8,52 @@ import config from '../Config';
 import aclUtils from '../utilities/aclUtils';
 import { pushMetric } from '../utapi/utilities';
 
+const { locationConstraints, restEndpoints, backends } = config;
+
+
+/**
+ * _checkLocationConstraint - check that a location constraint is explicitly
+ * set on the bucket and the value of the location is not listed in the
+ * locationConstraint config.
+ * Note: if data backend equals "multiple", you must set a location constraint
+ * @param {string} locationConstraint - location constraint
+ * @param {object} request - http request object
+ * @param {object} log - Werelogs logger
+ * @return {undefined}
+ */
+
+export function checkLocationConstraint(request, locationConstraint, log) {
+    // AWS JS SDK sends a request with locationConstraint us-east-1 if
+    // no locationConstraint provided.
+    const { parsedHost } = request;
+    let locationConstraintChecked;
+    if (locationConstraint) {
+        locationConstraintChecked = locationConstraint;
```

Comment:
what happened to check to make sure the locationConstraint is one listed in the config?